WcaOnRails/config/locales/de.yml

```diff
@@ -2344,16 +2349,17 @@ de:
       #original_hash: 72e2180
       regards_html: 'Viele Grüße, %{people}.'
       new:
-        #original_hash: a32db59
-        mail_subject: >-
-          Deine Anmeldung für %{comp_name} wurde übermittelt und die Bestätigung
-          steht noch aus
-        #original_hash: 8374583
-        registered_html: 'Du hast dich gerade für %{comp_name} registriert.'
-        #original_hash: 13fabed
+        #original_hash: 63591e5
+        mail_subject: 'Anmeldung Ausstehend: %{comp_name}'
```

Comment:
[CONTROVERSIAL]
Ich bin mit "ausstehend" nicht ganz glücklich, weil das nach meinem Bauchgefühl so klingt wie "Du hast da noch was nicht fertig abgesendet". Es lässt sich meiner Meinung nach zu einfach falsch verstehen als "Es steht noch aus, dass du deine Anmeldung fertig absendest".

Ich weiß, dass da im englischen Original "Registration Pending" steht, insofern ist die Übersetzung sicher nicht _falsch_. Es sollte aber eigentlich zum Ausdruck kommen, dass der*die Teilnehmer*in jetzt alles in seiner*ihrer Macht stehende **richtig** getan hat, das System alle Daten geschluckt hat, aber deswegen noch nicht die Hotels und Flüge gebucht werden können. Dafür sollte sich ein präziserer Ausdruck finden lassen.

Liegt vielleicht auch daran begründet, dass im Englischen die "Registration" sowohl der Akt des "sich-registrierens" als auch der Zustand des "fertig-registriert-seins" ausdrücken kann. Deswegen funktioniert die wörtliche Übersetzung "Registration Pending" dort besser.

Wie wäre es mit "Anmeldung in Überprüfung"? Bin auch für alternative Ideen offen.

Reply:
Das ist ein guter Punkt, irgendwie muss man zum Ausdruck bringen, dass da bald ein Mensch drüber schaut und prüft, dass auch evtl. Zahlungen eingegangen sind. Insofern ist der Vorschlag sinnvoll. Das gebe ich auch einfach noch an die anderen Delegates weiter, vielleicht gibt es da noch weitere Ideen oder Gesichtspunkte.